src/ert/ensemble_evaluator/snapshot.py

```diff
@@ -399,7 +445,7 @@ def get_successful_realizations(self) -> typing.List[int]:
         return [
             int(real_idx)
             for real_idx, real_data in self._my_partial._realization_states.items()
-            if real_data[ids.STATUS] == state.REALIZATION_STATE_FINISHED
+            if real_data["status"] == state.REALIZATION_STATE_FINISHED
```

Comment:
Should we remove these from identifiers.py if we don't need them? (assuming we don't need them because the TypedDict would complain if we set some illegal dict key.)

Reply:
we can use them as long as they are marked Final. typechecker will complain if not. did that as part of #8182